### ISSUE TITLE
User-input pop-up dialogue

### ISSUE THREAD

**Issue by Aya-Badawy89** [gama-platform/gama#3061]
In the dialogue that pops-up for user-input, can you please add the possibility to modify the font of the dialogue title?
For example, I want to use the space of the title to add info for the user that can guide his/her input. Therefore, I would like the font of the title to be bigger.
In general, it would be good if we can customize this dialogue. Please see the image attached.
![screenshot](https://user-images.githubusercontent.com/60944260/108714228-36402780-7511-11eb-931c-1c1b4ac624ac.png)

**Comment by hqnghi88** [comment]
Hi,
I just push a solution, you can customize with the font type in gaml.
	font font_default <- font("Helvetica", 38) const: true;
	map answers <- user_input("Address can be a pair lat,lon (e.g; '48.8566140,2.3522219')", [enter("Address", "")],font_default);
But it is only for gama  1.8.2 or if you use git base branch 1.8.2.

**Comment by Aya-Badawy89** [comment]
I downloaded GAMA 1.8.2 to workout this solution, but the versio does not open. It sends an error when I try to launch the application

**Comment by Aya-Badawy89** [comment]
I downloaded Gama 1.8.2 and tried running the code above, but it runs an error

**Comment by hqnghi88** [comment]
Sorry, forgot to release the binaries, it should be there in 15 mins. Please retry after that.

**Comment by Aya-Badawy89** [comment]
I tried again. I wrote the code _font_default <- font("Helvetica", 38) const: true;__
It says (unknown facet const). I tried working without the facet but then when running the input-dialogue appears empty with nothing written in it

**Comment by hqnghi88** [comment]
No idea which font on your machine, may be Arial?

**Comment by Aya-Badawy89** [comment]
I think it is not related to the type of font, but to the facet (const). It runs an error

**Comment by hqnghi88** [comment]
Tested with new download from 1.8.2 release 
![image](https://user-images.githubusercontent.com/6105384/109017524-d6e93f80-76e9-11eb-8d47-8918e239c469.png)

**Comment by Aya-Badawy89** [comment]
Yes, no error in writing the code. But when you run it, the dialogue becomes empty

![Screenshot 2021-02-24 14 53 06](https://user-images.githubusercontent.com/60944260/109018450-0a0edd00-76b0-11eb-8058-e4dea99e7867.png)

**Comment by hqnghi88** [comment]
which version did you try? It is only for gama 1.8.2 alpha 
![Uploading image.png…]()

**Comment by Aya-Badawy89** [comment]
Yes, I tried it on Gama 1.8.2 alpha and I have Java 15.
May be there is something wrong at my side. No worries. I will try to continue without changing the font

**Comment by AlexisDrogoul** [comment]
Has this issue been solved ? We have to think about passing more properties to these dialogs (font, color, background, etc.).

**Comment by AlexisDrogoul** [comment]
Hi

I've just tested this in relation with #3226 and it works well. I've also added the possibility to define a color for the background of the title, and I will commit this soon (probably tomorrow). 

Using `user_input_dialog(title, values, font)`
![Monosnap 2021-10-10 13-00-14](https://user-images.githubusercontent.com/579256/136684547-fe661d2c-9912-4705-af71-5970fc0e3aef.png)

Using `user_input_dialog(title, values, font, #yellow)`
![Monosnap 2021-10-10 13-20-45](https://user-images.githubusercontent.com/579256/136685029-7f820124-e3c4-462b-bc4b-8947b977ff53.png)

**Comment by Aya-Badawy89** [comment]
Hello Alexis,

Yes, the image of the error appears when I run the code which I am showing. I tried writing it with font('Helvetica Neue', 10, #bold) as you suggested but the error still there

**Comment by AlexisDrogoul** [comment]
Hi,

Could you attach here a very simple example where the popup dialog, not only does not use the custom font / color, but also fails to display correctly (or displays an error ? I'm not quite sure what's happening on your side) ? Just to make sure that we run the same tests. 

Thanks
Alexis
